bundles/org.eclipse.passage.loc.workbench/src/org/eclipse/passage/loc/internal/workbench/wizards/InnerClassifierWizard.java

```diff
@@ -73,13 +74,16 @@ protected void store(Optional<R> container, EObject candidate) {
 		if (!container.isPresent()) {
 			return;
 		}
-		EReference reference = candidate.eClass().getEAllReferences().stream()//
-				.filter(r -> r.isContainer())//
-				.findFirst()//
-				.get();
+		EReference reference = containerEReference(candidate.eClass()).get();
 		candidate.eSet(reference, container.get());
 		Resource resource = candidate.eResource();
 		Optional.ofNullable(resource).ifPresent(r -> LocWokbench.save(r));
 	}
 
+	private Optional<EReference> containerEReference(EClass eClass) {
+		return eClass.getEAllReferences().stream()//
+				.filter(r -> r.isContainer())//
```

Comment:
static reference `EReference::isContainer` should work

Reply:
done